.gitignore

```diff
@@ -15,8 +15,8 @@ Authomatic.egg-info/
 bootstrap/bootstrap.py
 bower_components/
 build/
-config.py
-config_public.py
+config.py*
+config_secret.py
```

Comment:
Maybe move to local .gitignore where the config file is expected and use a regex variant config*.py ?

Reply:
Historically the 'secret' file was called `config.py` - I did think about using this name to instead store the 'public' config - however this could create 2 risks - either a dev (who currently has their secrets in that file locally) commits it by accident, or overwrites it by pulling the public data.

Having an explicit 'public' and 'secret' in the file name also acts as a reminder to anyone (well, me!) committing new config data as to which file it should be in.